schema.sql

```diff
@@ -1,10 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    id INT,
+    id SERIAL PRIMARY KEY,
     name VARCHAR(100),
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOL,
-    weight_kg DECIMAL  
+    weight_kg DECIMAL,
+    species VARCHAR(100)  
```

Comment:
- [ ] Kindly add the `species` column using [`ALTER TABLE`](https://www.postgresql.org/docs/current/sql-altertable.html), instead of editing the `animals` table in the Schema file. This is because we are supposed to deal with the `animals` table as if it was already created and we need to modify the DB without removing it and creating it again.

Reply:
Ok, thank you.